app/views/users/update.js.erb

```diff
@@ -0,0 +1,9 @@
+<% if @user.errors.blank? %>
+  $('#setUserEmailModal').modal('hide');
+  element = $('td.col-name:contains("<%= @user.oim_id %>")').siblings('.col-email')
+  $(element).html('<%= @user.email %>')
+  $.ajax({url: '/users/password', type: 'POST', data: {user: { email: '<%= @user.email %>' }}, dataType: 'script'})
+<% else %>
```

Comment:
I would consider separating this into two views. update and error. Then have the conditional inside the controller - this makes it easier to mock